packages/client-core/src/world/Location.tsx

```diff
@@ -64,7 +60,11 @@ const LocationPage = ({ online }: Props) => {
   let [glassDisabled] = useFeatureFlags([FeatureFlags.Client.Glass])
   glassDisabled = glassDisabled && searchParams.get('glassUI') === null
 
-  useNetwork({ online })
+  const {
```

Comment:
this doen't make any sense, you are passing in values that are then used to set the same state